tests/api/items/inventory_view.test.js

```diff
@@ -31,6 +32,22 @@ describe('items:inventory-view', () => {
     res.itemsByDate.should.be.an.Array()
   })
 
+  it('should return an inventory-view for user items without shelf', async () => {
+    const user = await createUserWithItems()
+    const shelf = await createShelf(user)
+    let res = await publicReq('get', `${endpoint}&user=${user._id}&no-shelf=true`)
```

Comment:
As far as I can see, `createUserWithItems` does NOT create the indicated number of items (10), I haven't figured out why yet, but it sometimes create only one item which causes the `shelves?action=add-items` to fail.

Any idea on how to proceed on this point?

Reply:
@Aethelflaed that's weird, do you see any error in the test server logs? (fyi, you can follow those logs with `tail -f ./logs/test-server.log -n 100`)

I added a debug commit to https://github.com/inventaire/inventaire/tree/430-get-user-items-without-shelf

Reply:
@maxlath thanks, I don't know why I didn't check the server logs, the answer is quite easy indeed: we (as in my code but the previous inventory-view test also) make a public request on the inventory view but there's no guarantee that there's going to be any public items.

I slighlty changed createUserWithItems to ensure at least two public items